pkg/network/controllers/vm.go

```diff
@@ -165,17 +165,17 @@ func applyDynamicIfaceRequestOnVMI(
 }
 
 func clearDetachedInterfaces(
-	specIfaces []v1.Interface,
-	specNets []v1.Network,
-	statusIfaces map[string]v1.VirtualMachineInstanceNetworkInterface,
+	ifaces []v1.Interface,
+	nets []v1.Network,
+	ifaceStatusesByName map[string]v1.VirtualMachineInstanceNetworkInterface,
```

Comment:
nit: even that it is a map, it would say `ifaceStatusByName`
because it represent it good enough, and easier to grasp (or even to pronounce by heart when reading)
better to not express the type in the var name here imho

Reply:
> nit: even that it is a map, it would say `ifaceStatusByName` because it represent it good enough, and easier to grasp (or even to pronounce by heart when reading) better to not express the type in the var name here imho

The type name is `VirtualMachineInstanceNetworkInterface`, so it is not represented in the name.
This name is used all over the network code base so the intention here was to align it to existing code.